### PR TITLE
crowdstrike pt 1

### PR DESCRIPTION
**Pull request by calkim-panther**
### Background

unusual parent child detection
system log tampering
credential dumping tool
cryptomining detection

### CODE REVIEW

File: rules/crowdstrike_rules/crowdstrike_credential_dumping_tool.yml [hunk 1]
```diff
@@ -0,0 +1,166 @@
+AnalysisType: rule
+Description: Detects usage of tools commonly used for credential dumping.
+DisplayName: "Crowdstrike Credential Dumping Tool"
+Enabled: true
+Filename: crowdstrike_credential_dumping_tool.py
+Severity: Critical
+Tests:
+    - ExpectedResult: true
+      Log:
+        aid: 877761efa8db44d792ddc26bc2c71ce1
```
Comment (jzandona) [inline rules/crowdstrike_rules/crowdstrike_credential_dumping_tool.yml:10]: Should redact aid and cid

File: rules/crowdstrike_rules/crowdstrike_credential_dumping_tool.yml [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+            event_simpleName: ProcessRollup2
+            id: 081d64d7-17fb-40c0-8767-48ff1e2ee2dd
+            name: ProcessRollup2V19
+            timestamp: "1682106752722"
+        event_platform: Win
+        event_simplename: ProcessRollup2
+        fdr_event_type: ProcessRollup2
+        id: 081d64d7-17fb-40c0-8767-48ff1e2ee2dd
+        name: ProcessRollup2V19
+        p_any_ip_addresses:
```
Comment (andrea-youwakim) [inline rules/crowdstrike_rules/crowdstrike_credential_dumping_tool.yml:61]: any and all p_ values should be scrubbed from your test cases

File: rules/crowdstrike_rules/crowdstrike_credential_dumping_tool.yml [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
```
Comment (andrea-youwakim) [inline rules/crowdstrike_rules/crowdstrike_credential_dumping_tool.yml:138]: same comment as above

**Review by andrea-youwakim — COMMENTED** [review]
p_values removed from all test cases please

**Review by andrea-youwakim — APPROVED** [review]
extends the crowdstrike pack!

**Pull request merged**